roles/puppet_agent/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: manage puppet package
+  package:
+    name: "{{ puppet_package_name }}"
+    state: "{{ puppet_package_state }}"
+  when: puppet_package_manage
+
+- name: ensure puppet group
+  group:
+    name: "{{ puppet_group_name }}"
+    state: "{{ puppet_group_state }}"
+  when: puppet_group_manage
+
+- name: puppet ssl bootstrap
+  when: puppet_bootstrap_manage|bool
```

Comment:
you use `|bool` here, but not with the others? why?

Reply:
Likely a leftover because I wasn't sure if it was needed or not. Ansible variable handling is the worst.

Reply:
Fixed